.github/workflows/lint.yml

```diff
@@ -26,3 +26,11 @@ jobs:
 
       - name: Check playbooks with --syntax-check
         run: make syntax-check
+
+  ansible-lint:
```

Comment:
I had to use the marketplace action here instead of the makefile command because for some reason the github action wouldn't recognize the arguments I was passing in. i.e. it wouldn't recognize the `.ansible-lint` config file. 

This action properly recognizing the config file and is an official ansible action.